feat: transform optimize 支持判断 process.nextTick 及 process.browser

@ts-morph/common 的产物中基于 `process.nextTick` 及 `process.browser` 条件式引入 Node.js 的标准库（具体代码可访问 [unpkg.com](https://unpkg.com/@ts-morph/common@0.11.1/dist/typescript.js) 搜索 `process.` 查看），目前该 if 语句不会被删除所以构建会报错

报错示例：

```bash
Build failed.
fatal - [Error:
x Module not found: Can't resolve 'perf_hooks'
,-[../node_modules/_@ts-morph_common@0.11.1@@ts-morph/common/dist/typescript.js:3491:1]
3491 |         if (typeof process !== "undefined" && process.nextTick && !process.browser && typeof module === "object" && typeof require === "function") {
3492 |             try {
3493 |                 var performance_1;
3494 |                 var _a = require("perf_hooks"), nodePerformance_1 = _a.performance, PerformanceObserver_1 = _a.PerformanceObserver;
:                          ^^^^^^^^^^^^^^^^^^^^^
3495 |                 if (hasRequiredAPI(nodePerformance_1, PerformanceObserver_1)) {
3496 |                     performance_1 = nodePerformance_1;
3497 |                     // There is a bug in Node's performance.measure prior to 12.16.3/13.13.0 that does not
`----
] {
code: 'GenericFailure'
}
```

--- a/crates/mako/src/plugins/javascript.rs
+++ b/crates/mako/src/plugins/javascript.rs
@@ -224,16 +224,14 @@ pub fn is_import_meta_url(expr: &Expr) -> bool {
     matches!(
         expr,
         Expr::Member(MemberExpr {
-            obj:
-                box Expr::MetaProp(MetaPropExpr {
-                    kind: MetaPropKind::ImportMeta,
-                    ..
-                }),
-            prop:
-                MemberProp::Ident(Ident {
-                    sym: js_word!("url"),
-                    ..
-                }),
+            obj: box Expr::MetaProp(MetaPropExpr {
+                kind: MetaPropKind::ImportMeta,
+                ..
+            }),
+            prop: MemberProp::Ident(Ident {
+                sym: js_word!("url"),
+                ..
+            }),
             ..
         })
     )

--- a/crates/mako/src/transformers/transform_try_resolve.rs
+++ b/crates/mako/src/transformers/transform_try_resolve.rs
@@ -1,7 +1,7 @@
 use std::sync::Arc;
 
 use mako_core::swc_ecma_ast::{
-    AssignExpr, BlockStmt, CallExpr, Expr, ExprOrSpread, ExprStmt, Stmt, TryStmt,
+    AssignExpr, BlockStmt, CallExpr, Decl, Expr, ExprOrSpread, ExprStmt, Stmt, TryStmt,
 };
 use mako_core::swc_ecma_visit::{VisitMut, VisitMutWith};
 
@@ -64,6 +64,14 @@ impl VisitMut for TryResolve<'_> {
                             }),
                         ..
                     }) => self.handle_call_expr(call_expr),
+                    // e.g. var x = require('x');
+                    Stmt::Decl(Decl::Var(var_decl)) => {
+                        for decl in &mut var_decl.decls {
+                            if let Some(Expr::Call(call_expr)) = decl.init.as_deref_mut() {
+                                self.handle_call_expr(call_expr);
+                            }
+                        }
+                    }
                     _ => {}
                 }
             }

